packages/server/server/src/helpers/mock.ts

```diff
@@ -67,64 +76,93 @@ export const registerMockHandlers = async ({
   }
 
   if (!mockFilePath) {
-    return null;
+    return undefined;
   }
 
-  const { default: mockModule, config } = await import(mockFilePath);
+  const { default: mockHandlers, config } = (await import(
+    mockFilePath
+  )) as MockModule;
 
   const enable = config?.enable as
     | boolean
     | ((req: IncomingMessage, res: ServerResponse) => boolean)
     | undefined;
 
   if (enable === false) {
-    // eslint-disable-next-line consistent-return
-    return;
+    return undefined;
   }
 
-  if (!mockModule) {
+  if (!mockHandlers) {
     throw new Error(`Mock file ${mockFilePath} parsed failed!`);
   }
 
-  Object.entries(mockModule as MockConfig).forEach(([key, handler]) => {
+  return {
+    default: mockHandlers,
+    config,
+  };
+};
+
+export const getMatched = (request: InternalRequest, mockApis: MockAPI[]) => {
+  const { path: targetPathname, method: targetMethod } = request;
+
+  const matched = mockApis.find(mockApi => {
+    const { method, path: pathname } = mockApi;
+    if (method.toLowerCase() === targetMethod.toLowerCase()) {
+      return match(pathname, {
+        encode: encodeURI,
+        decode: decodeURIComponent,
+      })(targetPathname);
+    }
+
+    return false;
+  });
+
+  return matched;
+};
+
+export async function initOrUpdateMockMiddlewares(pwd: string) {
+  const mockModule = await getMockModule(pwd);
+
+  const mockHandlers = mockModule?.default;
```

Comment:
In the getMockModule function, it also require `module.default`, so does this need to get default property twice?